lib/Dialect/Torch/Transforms/GlobalizeObjectGraph.cpp

```diff
@@ -345,9 +347,15 @@ LogicalResult ObjectGraphGlobalizer::rewriteMethods() {
       FuncOp func = symbolTable.lookup<FuncOp>(method.function());
       if (failed(verifyMethodConformsToSubset(func)))
         return failure();
-      func.setVisibility(SymbolTable::Visibility::Public);
+      if (!method.isPrivate())
```

Comment:
i.e. I bet if you used the same visibility attributes, you could just copy here.

Reply:
see above comment.